src/main/java/hudson/plugins/git/extensions/impl/GitLFSPull.java

```diff
@@ -44,6 +47,12 @@ public void decorateCheckoutCommand(GitSCM scm, Run<?, ?> build, GitClient git,
         }
     }
 
+    @Override
+    public void determineSupportForJGit(GitSCM scm, UnsupportedCommand cmd) {
+        List<RemoteConfig> repos = scm.getRepositories();
+        cmd.lfsRemote(repos.get(0).getName());
```

Comment:
Since repos.get() returns a list of entries, shouldn't this iterate over the list?
```suggestion
        cmd.lfsRemote(repos.get(0).getName());
```